packages/common/react-components/src/props/Density.ts

```diff
@@ -0,0 +1,5 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+export type Density = 'coarse' | 'fine';
```

Comment:
I like that these are literal with respect to the quality of density :). Although I think `compact` and `full` (default) are a little less "abstract".

Reply:
That's fair, though I wanted to pick a continuum that 1) indicates a connection to interactive constraints and 2) can potentially map to Tailwind-style “t-shirt sizes”, e.g. if we were to introduce ‘extra’ values, to me ‘extra-coarse’ is more readily understood than ‘extra-full’.